packages/loaders/openapi/package.json

```diff
@@ -29,18 +29,22 @@
   },
   "dependencies": {
     "@graphql-mesh/cross-helpers": "0.2.2",
-    "@graphql-mesh/utils": "0.40.0",
     "@graphql-mesh/string-interpolation": "0.3.2",
     "@graphql-mesh/types": "0.80.2",
+    "@graphql-mesh/utils": "0.40.0",
     "@omnigraph/json-schema": "0.31.0",
+    "@types/cookie-parser": "1.4.3",
```

Comment:
I think those should go under devDependencies

Reply:
I actually intended to do so 😆